packages/vm/src/parsed_wasm.rs

```diff
@@ -147,4 +186,33 @@ impl<'a> ParsedWasm<'a> {
 
         Ok(this)
     }
+
+    /// Perform the expensive operation of validating each function body
+    ///
+    /// Note: This function caches the output of this function into the field `funcs_to_validate` so repeated invocations are cheap.
```

Comment:
This should say `func_validator`, right?

Reply:
It should. Nice catch